File: lemur-app/src/main/java/io/github/malczuuu/lemur/app/infra/data/jpa/player/PlayerEventLogJpaRepository.java
package io.github.malczuuu.lemur.app.infra.data.jpa.player;

import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PlayerEventLogJpaRepository extends JpaRepository<PlayerEventLogEntity, Long> {

  List<PlayerEventLogEntity> findAllByPlayerId(Long playerId);
}

File: lemur-app/src/main/kotlin/io/github/malczuuu/lemur/app/infra/data/jpa/player/PlayerEventLogJpaRepository.kt
package io.github.malczuuu.lemur.app.infra.data.jpa.player

import org.springframework.data.jpa.repository.JpaRepository

interface PlayerEventLogJpaRepository : JpaRepository<PlayerEventLogEntity, Long> {

    fun findAllByPlayerId(playerId: Long): List<PlayerEventLogEntity>
}
